src/main/java/com/mineboundteam/minebound/item/MyrialSword.java

```diff
@@ -0,0 +1,49 @@
+package com.mineboundteam.minebound.item;
+
+import com.mineboundteam.minebound.entity.MyrialSwordEntity;
+import com.mineboundteam.minebound.item.registry.ItemRegistry;
+import net.minecraft.world.InteractionHand;
+import net.minecraft.world.InteractionResultHolder;
+import net.minecraft.world.entity.player.Player;
+import net.minecraft.world.item.ItemStack;
+import net.minecraft.world.item.SwordItem;
+import net.minecraft.world.item.Tier;
+import net.minecraft.world.level.Level;
+import org.jetbrains.annotations.NotNull;
+
+public class MyrialSword extends SwordItem {
+    public MyrialSword(Tier tier, int attackDamageModifier, float attackSpeedModifier, Properties properties) {
+        super(tier, attackDamageModifier, attackSpeedModifier, properties);
+    }
+
+    @Override
+    public @NotNull InteractionResultHolder<ItemStack> use(@NotNull Level level, @NotNull Player player, @NotNull InteractionHand interactionHand) {
```

Comment:
Not sure if it's possible, but the intention was to have the sword track the user _while using_ it. The toggle is fine if the lerp for player look and the translation is too difficult at the moment; having a stationary spinning blade of death is cool too lol

Reply:
Movement is choppy, is there a way to smooth it?